src/main/java/ch/uzh/ifi/hase/soprafs22/service/InGameWebsocketService.java

```diff
@@ -89,7 +92,8 @@ public Move verifyMove(Game game, Move move, String username){
         // Add user details to move so that everybody knows who made the move
         User user = userRepository.findByUsername(username);
         if(this.checkHasNoCardsLeft(game, username)){
-            throw new ResponseStatusException(HttpStatus.NOT_FOUND, "User " + username + " has no cards left to make a move");
+            // TODO: Should this also just return null (move simply gets ignored)?
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND, username + " has no cards left to make a move");
```

Comment:
I'm actually not sure. I thought that exceptions are shown in the websocket messages but the client just does not react to them. 
I think we should stop throwing exceptions for websockets in the case that when errors thrown, it sends a message to every player logged in to our website.
Otherwise these errors just help us debug in the backend?

Reply:
Replaced it with a print and return null

Reply:
alright, also just feel free to resolve all conversations and merge as soon as you want.
my comments are just some 'tipps' 😄

Reply:
Haha sure, but I think they made sense plus I added the TODO for a reason